handler/handler.go

```diff
@@ -43,11 +44,11 @@ func TerraformProviderHandler(providerName string, config interface{}) (Handler,
 }
 
 // callNext should be called by a handler's Do() to call the next handler
-func callNext(nextH Handler, prevErr, err error) error {
+func callNext(ctx context.Context, nextH Handler, prevErr, err error) error {
 	nextErr := nextError(prevErr, err)
 
 	if nextH != nil {
-		return nextH.Do(nextErr)
+		return nextH.Do(ctx, nextErr)
```

Comment:
To make use of the context, it would need to check if the context has been cancelled before calling the next handler to determine if it should continue or stop further execution.

Reply:
I was focusing on just getting it plumbed through but that is an obvious case for it.
I'll add a check there.

Reply:
Thanks for looking into it. Context handling within the scope of the panos commit seems appropriate for the other PR, this one just stuck out to me if we could stop earlier.